libs/service-portal/graphql/src/lib/queries/delegations.ts

```diff
@@ -30,6 +32,10 @@ export const AUTH_DELEGATIONS_QUERY = gql`
         nationalId
         name
       }
+      from {
+        nationalId
+        name
+      }
```

Comment:
Maybe nice-to-have, but we should only query `to` for outgoing and `from` for incoming delegations. :/

Otherwise it may do 2x more national registry-calls than it needs to. Since we don't have DataLoader + caching on this field resolver.